server.js

```diff
@@ -27,11 +44,33 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
+
 // Start defining your routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/nominations', async (req, res) => {
+  const getNominations = await Nomination.find(req.query)
+
+  if (getNominations) {
+    res.json(getNominations)
+  } else {
+    res.status(404).json({ error: 'There is no nomination like this'})
+  }
+})
+
+app.get('/nominations/:year', async (req, res) => {
```

Comment:
`year` is not a good key for extracting single document, unfortunately. 

You need to use field which is always unique for every document, usually, it is `id` field.

Reply:
Hi Destiny! Jumping into the conversation here ;) There's no data on the deployed API! An idea for an endpoint that returns a single result can be to have params of year and category and then show the winner?

Reply:
Hi Matilda, 
Are you sure? I was just checking and I got back - here's one of the links https://golden-globe-api-mongo.herokuapp.com/nominees :)

Reply:
You're right Destiny, it seems to be deployed OK! Just fix so that the endpoint shows only one result!

Reply:
Fixed! :)